src/pages/Dashboard/MyAccount/MyAccountUi.tsx

```diff
@@ -93,7 +93,13 @@ export const MyAccountUi = ({
           </Tooltip>
         </div>
 
-        <Typography variant="h1" color="interactive.success" component="span">
+        <Typography
+          variant="h1"
+          color={
+            netApyPercentage && netApyPercentage > 0 ? 'interactive.success' : 'interactive.error'
```

Comment:
I think we should display 0 values in green. That's the default value for new users who haven't connected their wallet or supplied/borrowed any tokens yet, so the green color would be a bit more welcoming.
```suggestion
            netApyPercentage && netApyPercentage < 0 ? 'interactive.error' : 'interactive.success'
```